src/resources/SecurityCache/SecurityCacheInterfaces.ts

```diff
@@ -21,14 +27,13 @@ export interface SecurityCacheIdentityModel {
     provider?: string;
 }
 
-export interface SecurityProviderModelWithStatus {
+export interface SecurityProviderModel {
```

Comment:
There already seem to be an interface with that name in the [SourcesInterface file](https://github.com/coveo/platform-client/blob/master/src/resources/Sources/SourcesInterfaces.ts#L243), that's why the build fails:

```bash
11:46:41  src/resources/index.ts(23,1): error TS2308: Module './Sources' has already exported a member named 'SecurityProviderModel'. Consider explicitly re-exporting to resolve the ambiguity.
```

I believe the one you added here is correct, I suggest we remove the other one and fix the imports?

Reply:
I'll see what I can do!

Reply:
lgtm